content/authorization/application-registration-prerequisites.md

```diff
@@ -0,0 +1,207 @@
+
+- be assured that their data is being securely transmitted using modern transport layer security and encryption protocols
+- understand what legal entity they're sharing their data with
+
+These best practices should be considered as strongly-recommended prerequisites that software developers address during the application registration process, prior to making their applications available for general use.  Failure to comply with these prerequisites will result in Cerner describing your application as an "Unknown Application" to users during the OAuth authorization workflow.
+
+### Glossary of Terms
+
+* eTLD - Effective Top Level Domain is the base domain. eTLD values are defined in the [Public Suffix List](https://publicsuffix.org/list/public_suffix_list.dat).  Examples include: .com, .edu, .org, and .co.uk.
+* eTLD+1 - "Effective top level domain plus 1" describes a domain name that is one level deeper than the effective top level domain (eTLD). Examples include examplehealth.com, cerner.com and cerner.co.uk.
+* OV - Organization Validated - a certificate issuance process that involves the certificate authority validating the organization (business entity) requesting the certificate.
+* EV - Extended Validation - a certificate issuance process that involves the certificate authority performing extended validation of the organization (business entity) requesting the certificate.
+* DV - Domain Validated - a certificate issuance process that involves the certificate authority validating the requester owns the domain. No validation of the business entity is performed.
+* TLS - Transport Layer Security
+* DNS TXT record - a "text" record in the Domain Name System (DNS).  TXT records usually take form of name=value.
+
+### Technical Requirements
+
+The following technical requirements are stated using RFC-2119 terminology. These requirements focus on the security best practices that Cerner expects application developers to adhere to in order for Cerner to provide the best description of your application to end users:
+
+* Web applications MUST support the HTTPS 1.1 protocol.
+* Application SHOULD support TLS.  (Please refer to the Cerner CODE Program's TLS guidance document for further detailed practices related to TLS.)
+* Application SHOULD use (only) port 443 for TLS.
+* Application SHOULD support and prefer TLS 1.2 or higher.
+* Application SHOULD have a valid, non-expired TLS certificate issued from a trusted authority.
+* Application SHOULD use an OV or EV certificate, if possible, instead of a DV certificate.
+* Application's web site SHOULD host a DNS TXT record containing the Cerner-issued client identifier for the application (also referred to as a "system account ID" in CernerCentral or a "client identifier" in the [OAuth 2 specification](https://datatracker.ietf.org/doc/html/rfc6749#page-15) ).
```

Comment:
Please link to a section with how to setup DNS TXT record and the key, value that Cerner expects: "cerner-client-id={value}" 

the below text was from the section on how to query the DNS TXT record.
--- 
"cerner-client-id={value}" where value = Cerner-issued client identifier for your application
    in the case when multiple apps are registered for the same domain, cerner-client-ids will be stored in the same TXT record in the format:

    "cerner-client-id={value 1}"
    "cerner-client-id={value 2}"

    ...
    "cerner-client-id={value N}"

Reply:
@kolkheang For this "how to set up DNS TXT record" section, do you anticipate it saying something like "Consult you ISP's documentation for setting up a DNS TXT record." in addition to what we want them to put there?

Reply:
Correct. I think the main thing I'm concern about is the `cerner-client-id` key that they need to set with the value they need to set.

Reply:
I added a section to the main file.